qemu/tests/block_boot_multi_disks.py

```diff
@@ -0,0 +1,101 @@
+"""Test booting with multi disks"""
+import re
+
+from virttest import error_context, env_process, utils_misc, utils_disk
+from virttest.utils_misc import get_linux_drive_path
+from virttest.utils_windows.drive import get_disk_props_by_serial_number
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test booting with multi disks
+
+    1) Boot the vm with multi disk.
+    2) Check BIOS log for SEABios booting.
```

Comment:
2) Check warning message for SeaBIOS booting.
   The limit of virtio disks accessible to SeaBIOS is 16 in both RHEL8 and RHEL9.
   When the number of disks is greater than 16,  we should see a WARNING in the seabios log.
   No similar warning message in ovmf log.

Reply:
updated